contracts/settling_game/modules/travel/Travel.cairo

```diff
@@ -195,7 +195,7 @@ func get_travel_information{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, rang
 }
 
 @view
-func get_travel_time{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+func get_arrival_time{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
     traveller_coordinates: Point, destination_coordinates: Point
 ) -> (time: felt) {
     // get distance between two points
```

Comment:
It would be better to call the `get_travel_time` here so we don't duplicate the calculation.